seeds.md

```diff
@@ -0,0 +1,88 @@
+# Seeding the Application
+
+For development and testing purposes a developer may want to work with a large data set. If they were forced to achieve a large dataset by creating each record one-by-one it would waste valuable time and effort. The solution is [seeding the database](https://en.wikipedia.org/wiki/Database_seeding). To make this process even more effecient (_and fun_) we will be utilizing the [Faker](https://github.com/faker-ruby/faker) gem.
+
+## Step 1 - Installing Faker
+
+To start using `Faker`, first add it to your `Gemfile`: 
+```rb
+# bnb-library/Gemfile
+---------------------
+
+gem 'devise'
+gem 'trestle-auth'
+gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
+```
+Bundle your Gemfile: 
+```
+dip bundle install
+```
+We will setup a [rake task](https://docs.ruby-lang.org/en/2.2.0/Rake/Task.html) so that we can create all of our data in one reusable command.
+
+## Step 2 - Build a Rake Task
```

Comment:
Have we _ever_ done this? This seems out of place to me. I feel like we would typically just use the `seeds.rb` directly.

Reply:
Yes, I borrowed this from Jace's library app! I can move it back to the seeds, this is probably a unnecessary level of complexity.

Reply:
We'll leave that to #21